h/templates/layouts/group.html.jinja2

```diff
@@ -0,0 +1,20 @@
+{% extends "h:templates/layouts/base.html.jinja2" %}
+
+{% block page_title %}{{ page_title }}{% endblock %}
+
+{% block content %}
+  {{ panel('navbar') }}
+
+  <div class="content paper">
+    <div class="form">
```

Comment:
I suggest we get https://github.com/hypothesis/h/pull/3772 merged first and change the class name here to `form-container`